### ISSUE TITLE
Suggestion to scale out our community : Create an organisation managed by dominikh 

### ISSUE THREAD

**Issue by mikefaille** [dominikh/go-mode.el#183]
Hi @dominikh,
I need you to support this project even if you don't work full time on this project. I need you work as professionnal and scalling this community could be great !

Maybe, you could create a github organisation to manage this project and nominate maintainer.

Thank you for your answer,
Michael

**Comment by dominikh** [comment]
> I need you work as professionnal

I'd be happy to discuss pricing options for commercial support.

> Maybe, you could create a github organisation to manage this project and nominate maintainer.

Instead of proposing a solution, start by describing the problem. Which problem that currently exists are you trying to solve? Note that I am asking for actual problems, not hypothetical ones.

**Comment by mikefaille** [comment]
> I need you work as professionnal
My bad, sorry for my english.  I'm a professional who use your mode.
 
>  Which problem that currently exists are you trying to solve? 
There is many issues/PR open that could help your work.

**Comment by dominikh** [comment]
Creating an organisation wouldn't change that. Anyone is welcome to work on issues that affect them and send PRs; and I process PRs as time permits. If I wanted to, I could add collaborators to this repository, without the need for an organisation; but I don't want to. All decisions, especially code changes, would ultimately still have to go through me. Furthermore, making the organizational structure of the project more complex will waste more time on administration.

I realize that I haven't been investing terribly much time in go-mode recently. However, the project is overall in a pretty good spot. None of the open issues and PRs are all that important, they're mostly "nice to have" things, or about odd corner cases. Additionally, it's really a small project, not one that needs many maintainers. The project is perfectly usable on a day to day basis, and plenty of professionals are doing so.

tl;dr: you're always welcome to make contributions, but there's no need for an organisation or extra maintainers.

**Comment by mikefaille** [comment]
Sounds good ! :-)
Thank you for your answer.
